app/src/main/java/de/danoeh/antennapod/activity/AudioplayerActivity.java

```diff
@@ -569,6 +569,8 @@ protected void onPlaybackSpeedChange() {
 
     private void updateButPlaybackSpeed() {
         if (controller != null && controller.canSetPlaybackSpeed()) {
+            //Feed feed = <must somehow get feed here!!>
+            //butPlaybackSpeed.setText(feed.getPreferences().getCurrentPlaybackSpeed());
```

Comment:
But why? o_O
You need to set that speed in the PlaybackService anyway, so you could just ask it here.

So far, you could get the prefered playback speed from the feed and show it on the button, but the playback would still have a totally different speed...